providers/dynamodb_table.rb

```diff
@@ -1,5 +1,7 @@
 include AwsCookbook::DynamoDB
 
+provides :dynamo_db
```

Comment:
@gdavison wait… doesn’t this LWRP provide `dynamodb_table`? Are you adding an alias using this `provides` call?